app/views/track.js

```diff
@@ -146,6 +202,13 @@ const TrackView = Mn.LayoutView.extend({
             this.className += ' playlist-search';
         }
 
+        // Check if track is in selected
+        tracks = application.selected.get('tracks');
+        track = tracks.findWhere({ _id: this.model.get('_id') });
+        if (track) {
+            this.className += ' selected';
```

Comment:
ProTip: you can rely on [`aria-selected`](https://www.w3.org/TR/wai-aria/states_and_properties#aria-selected) to describe a _selected_ state for an element ;)